app/src/main/java/fr/free/nrw/commons/contributions/ContributionsListFragment.java

```diff
@@ -111,12 +114,14 @@ private void animateFAB(boolean isFabOpen) {
         this.isFabOpen = !isFabOpen;
         if (fabPlus.isShown()){
             if (isFabOpen) {
+                contributionsParentView.setAlpha((float) 1);
                 fabPlus.startAnimation(rotate_backward);
                 fabCamera.startAnimation(fab_close);
                 fabGallery.startAnimation(fab_close);
                 fabCamera.hide();
                 fabGallery.hide();
             } else {
+                contributionsParentView.setAlpha((float) 0.2);
```

Comment:
Would you mind adding two (here and the setAlpha above) small comments explaining very briefly the effect in each case? For instance "Fade the gallery when FAB is open" / etc
Thanks!

Reply:
@nicolas-raoul Added Comments :)